pipelines-scripts/release.ps1

```diff
@@ -11,9 +11,15 @@ param(
 # Output powershell version for debugging purposes and is probably generally good to know
 $PSVersionTable.PSVersion # Assuming powershell core (6)
 
-Write-Host "Installing Az module" # todo - should check if blueprint module is already installed
-Install-Module -Name Az.Blueprint -AllowClobber
-Write-Host "Successfully installed Az.Blueprint module"
+Write-Host "Installing Az module"
```

Comment:
If the client doesn't have the module installed, shouldn't it be installed rather than failing the script?

Reply:
Yes ... I forget the line of the installation as build.ps1 I will add it right now

Reply:
Yes right ! I'll fix it right now I forget to swipe thanks !

Reply:
This is working well - thanks for the change

Reply:
With pleasure. What about the other Todo below about the BlueprintId. Is it working ?